pkg/netpol/connlist/connlist.go

```diff
@@ -204,17 +206,57 @@ func (ca *ConnlistAnalyzer) ConnlistFromK8sCluster(clientset *kubernetes.Clients
 	return ca.getConnectionsList(pe)
 }
 
-// ConnectionsListToString returns a string of connections from list of Peer2PeerConnection objects
-func (ca *ConnlistAnalyzer) ConnectionsListToString(conns []Peer2PeerConnection) string {
+// ConnectionsListToString returns a string of connections from list of Peer2PeerConnection objects in the required output format
+func (ca *ConnlistAnalyzer) ConnectionsListToString(conns []Peer2PeerConnection, output string) (string, error) {
+	// supported output formats
+	const (
+		txtFormat  = "txt"
+		jsonFormat = "json"
+	)
+	// default if output parameter is empty
+	if output == "" {
+		output = txtFormat
+	}
+
+	if output == jsonFormat {
```

Comment:
may be better to validate format when getting command line parameters.

Reply:
done